cli/lsp/backend.rs

```diff
@@ -38,7 +38,7 @@ use super::tsc::TsServer;
 use super::utils;
 
 #[derive(Debug, Clone)]
-pub struct Backend {
+pub struct DenoLanguageServer {
```

Comment:
How about just `LanguageServer`?

Reply:
and then use `lspower::LanguageServer` for the trait?

Reply:
SGTM